pkg/apis/toolchain/v1alpha1/templateupdaterequest_types.go

```diff
@@ -34,22 +42,33 @@ type TemplateUpdateRequestSpec struct {
 // +k8s:openapi-gen=true
 type TemplateUpdateRequestStatus struct {
 	// Conditions is an array of current TemplateUpdateRequest conditions
-	// Supported condition types: ConditionReady
+	// Supported condition types: TemplateUpdateRequestComplete
 	// +optional
 	// +patchMergeKey=type
 	// +patchStrategy=merge
 	// +listType=map
 	// +listMapKey=type
 	Conditions []Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+
+	// SyncIndexes contains the `syncIndex` for each cluster in the MasterUserRecord.
+	// The values here are "captured" before the MasterUserRecord is updated, so we can
+	// track the update progress on the member clusters.
+	// +optional
+	// +patchStrategy=merge
+	// +mapType=atomic
+	SyncIndexes map[string]string `json:"syncIndexes,omitempty" patchStrategy:"merge"`
 }
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // TemplateUpdateRequest is the Schema for the templateupdaterequests API
 // +k8s:openapi-gen=true
+// +kubebuilder:printcolumn:name="Tier",type="string",JSONPath=`.spec.tierName`
+// +kubebuilder:printcolumn:name="Status",type="string",JSONPath=`.status.conditions[?(@.type=="Complete")].status`
+// +kubebuilder:printcolumn:name="Reason",type="string",JSONPath=`.status.conditions[?(@.type=="Complete")].reason`
 // +kubebuilder:resource:path=templateupdaterequests,scope=Namespaced
+// +kubebuilder:subresource:status
 // +kubebuilder:validation:XPreserveUnknownFields
-// +kubebuilder:printcolumn:name="Tier",type="string",JSONPath=`.spec.tierName`
 // +operator-sdk:gen-csv:customresourcedefinitions.displayName="Template UpdateRequest"
```

Comment:
isn't there missing space?
```suggestion
// +operator-sdk:gen-csv:customresourcedefinitions.displayName="Template Update Request"
```